source/slang-capture-replay/capture-manager.cpp

```diff
@@ -0,0 +1,53 @@
+
+#include <string>
+#include <sstream>
+#include <thread>
+#include "capture-manager.h"
+
+namespace SlangCapture
+{
+    CaptureManager::CaptureManager(uint64_t globalSessionHandle)
+        : m_encoder(&m_memoryStream)
+    {
+        std::stringstream ss;
+        ss << "gs-"<< globalSessionHandle <<"t-"<<std::this_thread::get_id() << ".cap";
+        m_fileStream = std::make_unique<FileOutputStream>(ss.str());
```

Comment:
This will create the file in a directory where the application started.
I am not sure if that will be an always available option.
We may need a way to specify the directory where the captured data gets stored.

Reply:
This is a temporary solution, I will think of some better way for the directory.
We fill definitely create a folder to store those capture files.

Reply:
That can be specified as a separate environment variable.